internal/pkg/build/sources/conveyorPacker_library.go

```diff
@@ -63,22 +65,52 @@ func (cp *LibraryConveyorPacker) Get(b *types.Bundle) (err error) {
 		return err
 	}
 
+	imagePath := ""
+
+	// Call for the 'SINGULARITY_NO_CACHE' in this package. Easy to change later if its all in a package.
+	// TODO: Maybe theres a better way to do this?
+	noCache := envconfig.NoCache()
+
 	imageName := uri.GetName("library://" + imageRef)
-	imagePath := cache.LibraryImage(libraryImage.Hash, imageName)
 
-	if exists, err := cache.LibraryImageExists(libraryImage.Hash, imageName); err != nil {
-		return fmt.Errorf("unable to check if %v exists: %v", imagePath, err)
-	} else if !exists {
-		sylog.Infof("Downloading library image")
+	if !noCache {
```

Comment:
Lets switch the order of these cases so that we can avoid a double negative, makes it easier to read imo